src/magic_folder/test/test_web.py

```diff
@@ -560,15 +486,13 @@ def test_list_folders(self, folders):
             False,
         )
 
-        # note that treq_for_folders() will end up creating a configuration here
-        config = load_global_configuration(basedir)
```

Comment:
This `load_global_configuration` call was the only reason we weren't using `create_testing_configuration` in these tests. By providing a version of `treq_for_folders` that also returns the config, we can get rid of this call, and switch to using `create_testing_configuration` in (the refactored) `treq_for_folders`, which is #333.

I made this change here, since I wanted to write another tests that uses the config to make assertions about the effect of calling the API. It also turns out that the new helper was useful in writing the CLI tests as well.